amivapi/settings.py

```diff
@@ -45,8 +45,10 @@
 
 DEFAULT_CONFIG_FILENAME = 'config.yaml'
 
-# Timeout for connections to the SMTP server
-SMTP_TIMEOUT = 3
+# SMTP server defaults
+SMTP_HOST = 'localhost'
+SMTP_PORT = 0
```

Comment:
port should probably default to 25 or 587

Reply:
Since we are using starttls, we should probably use 587, I agree